### ISSUE TITLE
Autocorrect errors with Lint/ImplicitStringConcatenation and Style/StringConcatenation

### ISSUE THREAD

**Issue by MatzFan** [rubocop/rubocop#13107]
## Expected behavior

I expected `rubocop -A` autocorrect to execute without errors.

## Actual behavior

2 errors were logged:
```
An error occurred while Lint/ImplicitStringConcatenation cop was inspecting /home/me/gitlab/file.rb:3:71.
An error occurred while Style/StringConcatenation cop was inspecting /home/me/gitlab/file.rb:3:71.
```
## Steps to reproduce the problem

With file.rb
```ruby
# frozen_string_literal: true

driver.bidi.send_cmd('script.addPreloadScript', "functionDeclaration": """() => { #{script} }""")
```
Run `rubocop --debug -A file.rb`

## RuboCop version
```
1.65.1 (using Parser 3.3.4.2, rubocop-ast 1.32.0, running on ruby 3.3.4) [x86_64-linux]
  - rubocop-minitest 0.35.1
  - rubocop-performance 1.21.1
```
## Debug log
```
For /home/me/gitlab: Default configuration from /home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/config/default.yml
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /home/me/gitlab/file.rb
An error occurred while Lint/ImplicitStringConcatenation cop was inspecting /home/me/gitlab/file.rb:3:71.
Parser::Source::TreeRewriter detected clobbering
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter.rb:427:in `trigger_policy'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter.rb:414:in `enforce_policy'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:210:in `call'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:210:in `block in check_fusible'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:208:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:208:in `check_fusible'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:188:in `analyse_hierarchy'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:112:in `place_in_hierarchy'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:132:in `block in combine_children'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:131:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:131:in `inject'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:131:in `combine_children'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:222:in `merge'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:105:in `do_combine'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter/action.rb:31:in `combine'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/parser-3.3.4.2/lib/parser/source/tree_rewriter.rb:143:in `merge!'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/base.rb:366:in `apply_correction'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/base.rb:445:in `attempt_correction'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/base.rb:426:in `use_corrector'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/base.rb:420:in `correct'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/base.rb:210:in `add_offense'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/lint/implicit_string_concatenation.rb:42:in `block in on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/lint/implicit_string_concatenation.rb:64:in `block in each_bad_cons'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/lint/implicit_string_concatenation.rb:54:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/lint/implicit_string_concatenation.rb:54:in `each_cons'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/lint/implicit_string_concatenation.rb:54:in `each_bad_cons'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/lint/implicit_string_concatenation.rb:33:in `on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:107:in `public_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:107:in `block (2 levels) in trigger_responding_cops'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:171:in `with_cop_error_handling'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:106:in `block in trigger_responding_cops'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:105:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:105:in `trigger_responding_cops'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:69:in `on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `block in on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:71:in `on_pair'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `block in on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:71:in `on_hash'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:159:in `block in on_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:156:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:156:in `each_with_index'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:156:in `on_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:71:in `on_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:20:in `walk'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:87:in `investigate'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/team.rb:164:in `investigate_partial'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/team.rb:95:in `investigate'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:349:in `block in inspect_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:348:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:348:in `flat_map'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:348:in `inspect_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:291:in `block in do_inspection_loop'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:325:in `block in iterate_until_no_changes'
<internal:kernel>:187:in `loop'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:318:in `iterate_until_no_changes'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:287:in `do_inspection_loop'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:168:in `block in file_offenses'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:193:in `file_offense_cache'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:167:in `file_offenses'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:158:in `process_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:139:in `block in each_inspected_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:138:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:138:in `reduce'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:138:in `each_inspected_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:124:in `inspect_files'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:77:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command.rb:11:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/environment.rb:18:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:122:in `run_command'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:129:in `execute_runners'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:51:in `block in run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:81:in `profile_if_needed'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:43:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/exe/rubocop:19:in `block in <top (required)>'
/home/me/.rbenv/versions/3.3.4/lib/ruby/3.3.0/benchmark.rb:313:in `realtime'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/exe/rubocop:19:in `<top (required)>'
/home/me/.rbenv/versions/3.3.4/bin/rubocop:25:in `load'
/home/me/.rbenv/versions/3.3.4/bin/rubocop:25:in `<main>'
An error occurred while Style/StringConcatenation cop was inspecting /home/me/gitlab/file.rb:3:71.
undefined method `end_pos' for nil
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/mixin/range_help.rb:33:in `contents_range'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/style/string_concatenation.rb:153:in `block in replacement'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/style/string_concatenation.rb:147:in `map'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/style/string_concatenation.rb:147:in `replacement'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/style/string_concatenation.rb:90:in `block in register_offense'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/base.rb:412:in `correct'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/base.rb:210:in `add_offense'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/style/string_concatenation.rb:87:in `register_offense'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/style/string_concatenation.rb:81:in `on_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:143:in `public_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:143:in `block (2 levels) in trigger_restricted_cops'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:171:in `with_cop_error_handling'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:142:in `block in trigger_restricted_cops'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:141:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:141:in `trigger_restricted_cops'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:70:in `on_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `block in on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:71:in `on_pair'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `block in on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:137:in `on_dstr'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:71:in `on_hash'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:159:in `block in on_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:156:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:156:in `each_with_index'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:156:in `on_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:71:in `on_send'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-ast-1.32.0/lib/rubocop/ast/traversal.rb:20:in `walk'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/commissioner.rb:87:in `investigate'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/team.rb:164:in `investigate_partial'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cop/team.rb:95:in `investigate'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:349:in `block in inspect_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:348:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:348:in `flat_map'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:348:in `inspect_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:291:in `block in do_inspection_loop'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:325:in `block in iterate_until_no_changes'
<internal:kernel>:187:in `loop'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:318:in `iterate_until_no_changes'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:287:in `do_inspection_loop'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:168:in `block in file_offenses'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:193:in `file_offense_cache'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:167:in `file_offenses'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:158:in `process_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:139:in `block in each_inspected_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:138:in `each'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:138:in `reduce'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:138:in `each_inspected_file'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:124:in `inspect_files'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/runner.rb:77:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/command.rb:11:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli/environment.rb:18:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:122:in `run_command'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:129:in `execute_runners'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:51:in `block in run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:81:in `profile_if_needed'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/lib/rubocop/cli.rb:43:in `run'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/exe/rubocop:19:in `block in <top (required)>'
/home/me/.rbenv/versions/3.3.4/lib/ruby/3.3.0/benchmark.rb:313:in `realtime'
/home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/exe/rubocop:19:in `<top (required)>'
/home/me/.rbenv/versions/3.3.4/bin/rubocop:25:in `load'
/home/me/.rbenv/versions/3.3.4/bin/rubocop:25:in `<main>'
W

Offenses:

file.rb:3:72: W: [Corrected] Lint/ImplicitStringConcatenation: Combine "" and "() => { #{script} }" into a single string literal, rather than using implicit string concatenation.
driver.bidi.send_cmd('script.addPreloadScript', "functionDeclaration": """() => { #{script} }""")
                                                                       ^^^^^^^^^^^^^^^^^^^^^^^
file.rb:3:72: C: [Corrected] Style/StringConcatenation: Prefer string interpolation to string concatenation.
driver.bidi.send_cmd('script.addPreloadScript', "functionDeclaration": "" + "() => { #{script} }" + "")
                                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
file.rb:3:72: C: [Corrected] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
driver.bidi.send_cmd('script.addPreloadScript', "functionDeclaration": """() => { #{script} }""")
                                                                       ^^
file.rb:3:77: W: [Corrected] Lint/ImplicitStringConcatenation: Combine "() => { #{script} }" and "" into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.
driver.bidi.send_cmd('script.addPreloadScript', "functionDeclaration": "" + "() => { #{script} }""")
                                                                            ^^^^^^^^^^^^^^^^^^^^^^^
file.rb:3:95: C: [Corrected] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
driver.bidi.send_cmd('script.addPreloadScript', "functionDeclaration": """() => { #{script} }""")
                                                                                              ^^
file.rb:3:98: C: [Corrected] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
driver.bidi.send_cmd('script.addPreloadScript', "functionDeclaration": "" + "() => { #{script} }""")
                                                                                                 ^^
file.rb:3:101: C: [Corrected] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
driver.bidi.send_cmd('script.addPreloadScript', "functionDeclaration": "" + "() => { #{script} }" + "")
                                                                                                    ^^

1 file inspected, 7 offenses detected, 7 offenses corrected

2 errors occurred:
An error occurred while Lint/ImplicitStringConcatenation cop was inspecting /home/me/gitlab/file.rb:3:71.
An error occurred while Style/StringConcatenation cop was inspecting /home/me/gitlab/file.rb:3:71.
configuration from /home/me/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/rubocop-1.65.1/config/default.yml
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.65.1 (using Parser 3.3.4.2, rubocop-ast 1.32.0, running on ruby 3.3.4) [x86_64-linux]
Finished in 0.44195283799490426 seconds
```
